### ISSUE TITLE
sequence item 1: expected str instance, BoundField found

### ISSUE THREAD

**Issue by Tjorriemorrie** [sveetch/crispy-forms-foundation#48]
`forms.py`

```
class NameForm(forms.Form):
    your_name = forms.CharField(label='Your name', max_length=100)

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.helper = FormHelper()
        self.helper.form_action = 'POST'
```

`views.py`
```
'form': forms.NameForm(),
```

template
```
{% load crispy_forms_tags %}
{% crispy form %}
```

I get the error:
sequence item 1: expected str instance, BoundField found

>site-packages/crispy_forms_foundation/templates/foundation-6/field.html, error at line 22

**Comment by bluespider42** [comment]
Did you find a solution to this?

**Comment by bluespider42** [comment]
I fixed this by changing CrispyFoundationFieldNode.render to return `str(field)` instead of `field`. This matches the render method of the most recent release of crispy_forms.  
 [Line 88](https://github.com/sveetch/crispy-forms-foundation/blob/1e8c5dad9d8e3a3a6c3e114575c157c8e885ba37/crispy_forms_foundation/templatetags/crispy_forms_foundation_field.py#L88)

**Comment by sprda** [comment]
I believe this issue can be closed as it was fixed in release 0.9.0

**Comment by sveetch** [comment]
Yeah it have been fixed in https://github.com/sveetch/crispy-forms-foundation/commit/541a76ac9b786760f7588c93d29ab33d56354bc3 i forgot to close this issue
